_posts/2018-10-04-js-testing.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title:  "Javascript Projects Testing"
+date:   2018-10-04 12:00:00 -0400
+categories: software testing js javascript jest enzyme
+author: Tania Paiva
+---
+
+Should I test this? the answer is a BIG YES. Tests are your friends, and saviors sometimes (often).
+
+At Forward Financing, we feel proud when our projects reach high test coverage. It's our best insurance  
+for making improvements and upgrades later with the warranty that existing functionality will  
+keep working.
+
+Mistakes can always happen, and that is where tests can be real saviors, as an alert that  
+something was altered by mistake, or that some functionality is not providing the expected result.
+
+Tests verify the application meets the requeriments, it guarantees the code makes what we expect  
+it to do.
+
+Okay, tests can do a lot for us, but is not always sun and butterflies, writing good tests takes  
+time and it gets better with some experience.
+
+![Coverage]({{"/assets/test_coverage.png" | absolute_url}})
+
+In this article we will talk about the testing tools we use for our javascript projects.
```

Comment:
I think you can be more affirmative here, so you can skip the set up about "In this article..."

ie: "Here are some of the testing tools we use for our JavaScript projects" or something to that degree